types/types.go

```diff
@@ -58,13 +59,20 @@ type NotificationTypeID int
 // StateID represents ID value in `states` table.
 type StateID int
 
+// EventTarget matches the backend int code in the database where the notifications are sent (CCXDEV-8767)
+type EventTarget int8
```

Comment:
uint8 probably?

Reply:
I thought about using that type, but I'm not sure. Maybe we use `-1` for something in the future. I can put `uint8` there, not a big deal. What should I do?

Reply:
really does not matter :-)